release_dashboard/lib/widgets/common/continue_button.dart

```diff
@@ -0,0 +1,72 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
```

Comment:
This file is a copy paste from this PR: https://github.com/flutter/cocoon/pull/1494

Since that PR hasn't been approved, I have just copied here to unblock myself, and it will be rebased.

Reply:
In the future, prefer cherry-picking commits from the other PRs. For more complicated PRs, it's common for reviewers to look at specific commits instead of the entire history.